src/main/java/prefs/UpdateConfig.java

```diff
@@ -0,0 +1,35 @@
+package prefs;
+
+import util.Version;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains configuration information for updating
+ */
+public class UpdateConfig {
+
+    private boolean lastUpdateDownloadStatus;
```

Comment:
If the last download was successful. So if the last download was cut off halfway (connection lost or user close the HT suddenly), this will be `false`.